src/rules/unit-blacklist/README.md

```diff
@@ -49,3 +49,48 @@ a { height: 100vmin; }
 ```css
 a { animation: animation-name 5s ease; }
 ```
+
+## Optional Options
```

Comment:
Please double-check against the current documetnation that this is the title we're using.